File: clients/java/examples/client-get-astra-admin.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.admin.AstraDBAdmin;

public class Example {
  public static void main(String[] args) {
    // Get an admin object
    DataAPIClient client = new DataAPIClient("**APPLICATION_TOKEN**");
    AstraDBAdmin admin = client.getAdmin();
  }
}

File: clients/go/examples/client-get-astra-admin.go
package main

import (
	"log"

	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	client := astra.NewClient(
		options.API().SetToken("**APPLICATION_TOKEN**"),
	)

	_, err := client.Admin(options.API())

	if err != nil {
		log.Fatal(err)
	}
}
